Constellation/Node/Api.hs

```diff
@@ -113,6 +139,38 @@ data Whitelist = Whitelist
     , wlIPv6 :: Set HostAddress6
     } deriving Show
 
+decodeSendRaw :: BL.ByteString -> RequestHeaders -> Either String Send
+decodeSendRaw b h = case getHeaders [hContentLength, hFrom, hTo] h of
+    Right headers -> Right Send
+                    { sreqPayload = decodePayload (hmap ! hContentLength) b
+                    , sreqFrom    = mustDecodeB64PublicKey $ hmap ! hFrom
```

Comment:
"from" and "to" should just be string literals rather than hFrom/hTo vars (maybe if they were used in more places... more confusing than it's worth atm)